app/views/roles/_details.js.haml

```diff
@@ -0,0 +1,3 @@
+- if @type
+  $('#role_info').html('#{escape_javascript(render('info'))}');
+  = render_extensions :details
```

Comment:
Had to make this into a partial, as details should be loaded whenever group or role is changed.